config/environment.js

```diff
@@ -1,5 +1,9 @@
 /* eslint-env node */
 
+function isTruthy(val) {
+    return val.toString().toLowerCase() === 'true';
+}
```

Comment:
🙏

Reply:
[`isTruthy(1)`](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/.travis.yml#L44-L45)?

Reply:
All env vars become strings in node, but local.js can be any

Reply:
This works with booleans or strings. If you decide to use `1` then you get to feel bad and watch things not work.

Reply:
that is correct 🥇 

Either this function should be updated to accept other truthy values like `1`, or the places we use `1` should be changed to `true` and we should announce that `1` doesn't work anymore, in case anyone has it in their local.js.

Reply:
could also dump using environment variables altogether (i.e. only local.js) and save the headache

also, you can use environment variables in your local.js, if you please